pkg/session/serve.go

```diff
@@ -126,7 +130,24 @@ func (s *Session) serve() {
 
 		case "delete":
 			go s.delete()
-
+		case "logout":
+			stateFile, err := config.DefaultStateFile()
+			if err != nil {
+				log.Logger.Errorw("failed to get state file", "error", err)
+				break
+			}
+			dbRW, err := sqlite.Open(stateFile)
+			if err != nil {
+				log.Logger.Errorw("failed to open state file", "error", err)
+				break
+			}
+			if err := gpudstate.DeleteLoginInfo(ctx, dbRW); err != nil {
+				log.Logger.Errorw("failed to delete login info", "error", err)
+			}
+			dbRW.Close()
+			if _, err = exec.CommandContext(ctx, "systemctl", "stop", "gpud").Output(); err != nil {
+				log.Logger.Errorw("failed to stop gpud service", "error", err)
+			}
```

Comment:
shall we delay exec this command like reboot?

Reply:
yea we can do that..lemme update
initially thought since we are going to stop gpud anyway so prob do not need to wait for resp